Revision: lvm--dilinger--2--patch-3
Archive: dilinger@voxel.net--2004
Creator: Andres Salomon <dilinger@voxel.net>
Date: Mon May  3 02:03:30 EDT 2004
Standard-date: 2004-05-03 06:03:30 GMT
Modified-files: make.tmpl.in
New-patches: dilinger@voxel.net--2004/lvm--dilinger--2--patch-3
Summary: additional build system clean fix (distclean this time)
Keywords: 

* added files

    {arch}/lvm/lvm--dilinger/lvm--dilinger--2/dilinger@voxel.net--2004/patch-log/patch-3

* modified files

--- orig/make.tmpl.in
+++ mod/make.tmpl.in
@@ -174,7 +174,7 @@
 	      $(SOURCES:%.c=%.pot) $(LDDEPS) \
 	      config.cache config.log config.status \
 	      Makefile make.tmpl core \
-	      version.h lvm2.po
+	      version.h lvm.o
 
 .export.sym: .exported_symbols
 	set -e; (echo "Base {"; echo "	global:"; \



